### ISSUE TITLE
Is Fragment.setMenuVisibility supported?

### ISSUE THREAD

**Issue by httpdispatch** [Prototik/HoloEverywhere#378]
Hello

I'm trying to use viewpager with holoeverywhere fragments using FragmentPagerAdapter. Each fragment may have own menu. To adjust menu visibility FragmentPagerAdapter uses next code 

``` java
fragment.setMenuVisibility(false);
fragment.setUserVisibleHint(false);
```

But when i start application i can see menu from non visible fragments (left and right from current) (such as they are in attached state and seems it ignores setMenuVisibility). 
Can you please show me the right way of how to work with fragments menu in viewpager?

**Comment by Prototik** [comment]
For my application it's work. But i'm use FragmentStatePagerAdapter, but i don't think that it's important.

**Comment by httpdispatch** [comment]
Thanks, for the answer. Let me try again

**Comment by httpdispatch** [comment]
Tested on clean project and it is working fine. So i need to find problem in my code. Thanks for the help
